iop/cdvd/cdfs/src/cdfs_iop.h

```diff
@@ -2,7 +2,7 @@
 #define _CDFS_IOP_H
 
 #ifdef DEBUG
-#define DPRINTF(args...)	printf(args)
+#define DPRINTF(args...)	printf("CDFS: "args)
```

Comment:
why not use?
```
#define MODNAME "CDFS"
IRX_ID(MODNAME, 1, 1);

#define M_PRINTF(format, args...) \
    printf(MODNAME ": " format, ##args)
```
like here:
https://github.com/ps2dev/ps2sdk/blob/master/iop/dev9/atad/src/ps2atad.c#L40

Reply:
I thought that the string passed to IRX_ID() macro had to remain EXACTLY the same...

If I'm wrong, and that can be done, please let me know

Reply:
so...
you want me to use modname on all of them?

i would like to keep it this way

Reply:
I think that using @AKuHAK approach will be a better solution